### PR TITLE
Fix issue 72: Support multi-line assignment indenting on return statements.

### PR DESCRIPTION
**Pull request by lewis6991**
@vhda merge this if you are happy with it.

### CODE REVIEW

**Comment by lewis6991** [comment]
Rebased

**Comment by lewis6991** [comment]
@vhda any problems with this?

**Comment by vhda** [comment]
It's the bigger one, so I left it for last! :)

Let me take a look.

File: indent/verilog_systemverilog.vim [hunk 1]
```diff
@@ -258,16 +258,22 @@ function! s:GetContextIndent()
...
+          " the indentation of the open statement.
           let l:assign_offset = len(l:assign)
-          call verilog_systemverilog#Verbose("Increasing indent for an open assignment with values (by " . l:assign_offset .").")
+          call verilog_systemverilog#Verbose(
+            "Increasing indent for an open assignment with values (by " . l:assign_offset .")."
+          )
         else
-          " If the assignment is empty, simply increment the indent by one level
+          " If the assignment is empty, simply increment the indent by one
+          " level.
```
Comment (vhda) [inline indent/verilog_systemverilog.vim:272]: The comment you added below is bigger than this one. Why enforce a line break because of one word?

**Comment by vhda** [comment]
Should we start enforcing a specific textwidth for comments?

**Comment by lewis6991** [comment]
Maybe, I generally try to have it less than 80 chars whenever possible, though there are cases where it's unavoidable.

**Comment by vhda** [comment]
Github's diff syntax highlight failed to identify the string and I misread the patch. Seeing it now in vim I understand what you are doing.

I'm used to working with longer lines, but have no problems in limiting myself to 80 character.
Rebase your patch and I'll merge it.

**Comment by lewis6991** [comment]
rebased

**Pull request merged**